test/e2e_node/critical_pod_test.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 const (
-	criticalPodName            = "critical-pod"
+	criticalPodName            = "static-critical-pod"
```

Comment:
What is the reason for calling it static?

Reply:
criticalPodName call static-critical-pod because using annotation `"kubernetes.io/config.source"`to mark it.